release_notes/Service_Fabric_ReleaseNotes_91.md

```diff
@@ -14,59 +14,66 @@ The following packages and versions are part of this release:
-There are two ways to identify and confirm the issue
-
-A. If periodic backups were happening on any partition, it should be visible on Service Fabric Explorer(SFX) under Cluster->Application->Service->Partition->Backup. Here list of all backups being taken with creation time is available. Using this info and upgrade time, customer can identify whether a backup policy was enabled, backups were happening before upgrade and whether backups are happening post upgrade.
-
-B. Another way of checking and enumerating backups is calling this API [get partition backup list](https://learn.microsoft.com/en-us/rest/api/servicefabric/sfclient-api-getpartitionbackuplist).
-
-**Mitigation:**
-To mitigate, customers need to update the existing policy after upgrading to 9.1.1390.9590. User can call update API for Backup policy as mentioned in this doc [Update Backup Policy](https://learn.microsoft.com/en-us/rest/api/servicefabric/sfclient-api-updatebackuppolicy) with existing policy values. It will update the policy model inside BRS with new data model and BRS will start taking periodic backups again.
-
-**Steps:**
-1. Check and confirm issue as mentioned in "Identifying the issue" section above.
-2. If the issue is confirmed, update the backup policy with same old values by calling update API for Backup policy. Below is one sample -
-    ```powershell
-     $BackupPolicy=@{
-      Name = "DailyAzureBackupPolicy"
-      AutoRestoreOnDataLoss = "false"
-      MaxIncrementalBackups = "3"
-      Schedule = @{
-        ScheduleKind = "FrequencyBased"
-        Interval = "PT3M"
-      }
-      Storage = @{
-        StorageKind = "AzureBlobStore"
-        FriendlyName = "Azure_Storage_Sample"
-        ConnectionString = "<connection string values>"
-        ContainerName = "<Container Name>"
-      }
-      RetentionPolicy = @{
-        RetentionPolicyType = "Basic"
-        MinimumNumberOfBackups = "20"
-        RetentionDuration = "P3M"
-      }
-     }
-      $body = (ConvertTo-Json $BackupPolicy)
-      $url = 'https://<ClusterEndPoint>:19080/BackupRestore/BackupPolicies/DailyAzureBackupPolicy/$/Update?api-version=6.4'
-      Invoke-WebRequest -Uri $url -Method Post -Body $body -ContentType 'application/json' -CertificateThumbprint '<Thumbprint>'
-      # User should update the name of backup policy [DailyAzureBackupPolicy being used here and other possible values accordingly].
+* **Breaking Changes with FabricDNS:** 
+    Service Fabric DNS names fail to resolve in Process-based services for Windows clusters after upgrading to 9.1.1390.9590 when Hosting.DnsServerListTwoIps set to   true in Cluster settings.
+
+  **Identifying the issue:** In Service Fabric Explorer, check if there is a warning message of "System.FabricDnsService reported Warning for property 'Environment.IPv4'. FabricDnsService is not preferred DNS server on the node."
+
+  **Mitigation:** Set Hosting.DnsServerListTwoIps to false in the Service Fabric cluster settings or rollback cluster version.
```

Comment:
Why is there a new Breaking Change being added to the 9.1 release at this point?